web/js/data/map.js

```diff
@@ -47,37 +47,30 @@ export function dataMap(model, maps, config) {
       image = 'images/data.plus-button.png';
     }
 
-    if (feature !== hovering) {
-      return [new OlStyleStyle({
-        image: new OlStyleIcon({
-          src: image,
-          scale: dim.scale
-        })
-      })];
-    } else {
-      var offset = -(dim.size / 2.0 + 14);
-      return [new OlStyleStyle({
-        image: new OlStyleIcon({
-          src: image,
-          scale: dim.scale
-        }),
-        text: new OlStyleText({
-          font: 'bold 14px ‘Lucida Sans’, Arial, Sans-Serif',
-          text: feature.granule.label,
-          fill: new OlStyleFill({
-            color: '#ffffff'
-          }),
-          stroke: new OlStyleStroke({
-            color: 'rgba(0, 0, 0, .7)',
-            width: 5
-          }),
-          offsetY: offset
-        })
-      })];
-    }
+    return [new OlStyleStyle({
+      image: new OlStyleIcon({
+        src: image,
+        scale: dim.scale
+      })
+    })];
   };
 
   var hoverStyle = function (feature) {
+    var dim = getButtonDimensions();
```

Comment:
Since you're creating this variable in a new place, it might be a good time to take the opportunity and rename all of the `dim` variables in this file to `dimensions` so it's easier to read in the future. Your call.